docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
+The `load_assets_from_dbt_project` function will compile your dbt project, parse the metadata that dbt provides, and generate a set of software-defined assets that reflect the models in the project. These assets will all share the same underlying operation, which will invoke dbt to run the models represented by the loaded assets.
 
-Note that in both cases, the `models` option takes in a list of strings. The string `"tag:staging"` uses [dbt's node selection syntax](https://docs.getdbt.com/reference/node-selection/syntax) to filter models with the tag `"staging"`. For more details, [visit the official dbt documentation on the node selection syntax](https://docs.getdbt.com/reference/node-selection/syntax).
+### Loading `dbt` models from a dbt manifest.json file
 
-##### **... with configuration**
+For larger projects, the overhead involved in recompiling your entire project whenever your repository needs to be loaded may become a concern. In these cases, it's possible to load
 
-If you know what models you want to select ahead of time, you might prefer specifying this while configuring your resource. Because you aren't specifying any specific arguments at runtime, you can use the prebuilt `dbt_run_op`, instead of writing your own.
+```python
+import json
+from dagster_dbt import load_assets_from_dbt_manifest
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run_specific_models_preconfig endbefore=end_marker_dbt_cli_run_specific_models_preconfig dedent=4
-from dagster_dbt import dbt_cli_resource, dbt_run_op
-
-from dagster import job
-
-my_dbt_resource = dbt_cli_resource.configured(
-    {"project_dir": "path/to/dbt/project", "models": ["tag:staging"]}
+dbt_assets = load_assets_from_dbt_manifest(
+    json.load("path/to/dbt/manifest.json", encoding="utf8"),
 )
-
-@job(resource_defs={"dbt": my_dbt_resource})
-def my_dbt_job():
-    dbt_run_op()
 ```
 
-##### **... supplying an argument**
+Note: if you make any changes to your dbt project that change the structure of the project (such as changing the dependencies of a model or adding a new one), you'll need to regenerate your manifest file for those changes to be reflected in Dagster.
 
-If you want to change which models you select depending on what happens during execution, you should create your own op that invokes the dbt resource (rather than using a prebuilt op). This allows you to explicitly set the `models` parameter at runtime. This would look similar to the following:
+## Adding a resource
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run_specific_models_runtime endbefore=end_marker_dbt_cli_run_specific_models_runtime dedent=4
-from dagster import op
+In order to function, these assets require a "dbt" resource, which will be responsible for firing off dbt cli commands. The `dagster-dbt` integration provides the `dbt_cli_resource` to make this easy. This resource can be configured with CLI flags that will be passed i
```

Comment:
> In order to function, these assets require a "dbt" resource, which will be responsible for firing off dbt cli commands.

1. Are the quotes for `"dbt"` necessary? What's the purpose of having them here?
2. As CLI is an acronym, I feel like it should be in all caps. Is there a reason it wouldn't be?

Reply:
I put the quotes there to hint that the resource itself needs to be named "dbt", but I don't think it's necessary so I'll remove them. No real reason to make cli lowercase either